Documentation/CRDs/Cluster/external-cluster/advance-external.md

```diff
@@ -0,0 +1,68 @@
+# NFS storage
```

Comment:
Let's have a top-level title on each page, then all topics should be second-level titles.
```suggestion
# External Cluster Options

## NFS storage
```